### ISSUE TITLE
External pages not rendering in label generated menu

### ISSUE THREAD

**Issue by adamjohnson** [wvuweb/cleanslate-cms#82]
_From @dustinmazon on December 12, 2014 16:43_

So a while back Nathan put together a dynamic menu for me that was created from pages with the same label. The menu works in the edit mode in CleanSlate... but once you look at it in production view only the content pages populate the menu, the external linked pages do not load.

I am using this menu on the **Graduate Education** site in CleanSlate. The theme is called **graduate-education**. You can find this example in the following directory:

`views/layouts/_header__auxillary-nav.html`

**Example:**

```
<r:root>
    <ul>
		<r:children:each labels="top-navigation">
			<li>
				<a href="<r:page:url />"><r:page:name /></a>
			</li>
		</r:children:each>
	</ul>
</r:root>
```

**Screenshot of edit view:**

![screen shot 2014-12-12 at 11 21 31 am](https://cloud.githubusercontent.com/assets/1126703/5415236/c101639c-81f3-11e4-8097-de052d02d537.png)

**Screenshot of production view:**

![screen shot 2014-12-12 at 11 21 41 am](https://cloud.githubusercontent.com/assets/1126703/5415231/bdd1752c-81f3-11e4-8478-d1e3a50d34fb.png)

The link that I am trying to display is the FAQ, notice in the production view it does not render. Any thoughts on how to set the menu to display externally linked pages as well as normal content pages in the production view?

_Copied from original issue: wvuweb/cleanslate#81_

**Comment by adamjohnson** [comment]
_From @nreckart on December 12, 2014 18:39_

The FAQ page is not published. Add a published date in the properties and it will show.

**Comment by adamjohnson** [comment]
_From @dustinmazon on December 12, 2014 18:43_

Ok, so would there be a way that we could automatically assign a published date when an externally linked page is created.

**Comment by adamjohnson** [comment]
_From @nreckart on December 12, 2014 18:50_

So you're saying External Link pages should **_always_** be published immediately upon creation?

**Comment by adamjohnson** [comment]
_From @zeroedin on December 12, 2014 19:12_

I would lean first to say external pages should be automatically published as if you have the link to a page it's live.  

But I could also see a use case when that might not be the case.  Say site X has a page going live (like a blog article) that is embargoed.  Site Z wants to link to it, so they externally link it, then embargo the published on date, then the page would only show after the published on date as passed?  But say they don't immediately know the published on date and want to keep it hidden until they can fill it out.

**Comment by adamjohnson** [comment]
_From @nreckart on December 12, 2014 19:48_

Made External Link pages published by default on creation in 4a6c6f91fbf0d0f874.
